sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/placement/MarathonConstraintParser.java

```diff
@@ -44,58 +44,45 @@ private MarathonConstraintParser() {
     }
 
     /**
```

Comment:
This should probably be kept around in some form, it's expected to be used in frameworks that want to combine some hardcoded rules (in a service's Main.java) with user-entered rules.

For example HDFS could be exposing a user-visible setting to place all of the HDFS service in a particular location, and then using this to AND that setting against the hardcoded per-type rules.